src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveClient.kt

```diff
@@ -14,21 +17,32 @@ import java.net.URI
 
 @Component
 class OppgaveClient(
-    @Qualifier("azure") restOperations: RestOperations,
-    @Value("\${FAMILIE_INTEGRASJONER_API_URL}") private val integrasjonUrl: String,
+        @Qualifier("azure") restOperations: RestOperations,
+        @Value("\${FAMILIE_INTEGRASJONER_API_URL}") private val integrasjonUrl: String,
 ) : AbstractRestClient(restOperations, "familie.integrasjoner") {
 
     val oppgaveUrl = "$integrasjonUrl/api/oppgave"
 
+    fun hentOppgaver(finnOppgaveRequest: FinnOppgaveRequest): List<Oppgave>? {
+        val opprettOppgaveUri = URI.create("$oppgaveUrl/oppgave/v4")
+        val response =
+                postForEntity<Ressurs<FinnOppgaveResponseDto>>(
+                        opprettOppgaveUri,
+                        finnOppgaveRequest,
+                        HttpHeaders().medContentTypeJsonUTF8()
+                )
+        return response.data?.oppgaver
```

Comment:
Vet egentlige ikke om https://github.com/navikt/familie-ef-sak/blob/master/src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveClient.kt#L89 trengs, men ellers kan man bruke `getDataOrThrow()` i stedet for `.data?`

Denne endepunkten skal egentligen alltid returnere data hvis det ikke feiler (husk å oppdatere nullsjekk en nivå opp og)